### PR TITLE
OIDC integration

### PR DESCRIPTION
**Pull request by corest**

### CODE REVIEW

**Review by oponder — APPROVED** [review]

File: main.go [hunk 1]
```diff
@@ -99,6 +99,11 @@ func main() {
 
 	daemonCommand := newCommand.DaemonCommand().CobraCommand()
 
+	daemonCommand.PersistentFlags().String(f.Service.Installation.Guest.Kubernetes.API.Auth.Provider.OIDC.ClientID, "", "OIDC authorization provider ClientID.")
+	daemonCommand.PersistentFlags().String(f.Service.Installation.Guest.Kubernetes.API.Auth.Provider.OIDC.IssuerURL, "", "OIDC authorization provider IssuerURL.")
+	daemonCommand.PersistentFlags().String(f.Service.Installation.Guest.Kubernetes.API.Auth.Provider.OIDC.UsernameClaim, "", "OIDC authorization provider UsernameClaim.")
+	daemonCommand.PersistentFlags().String(f.Service.Installation.Guest.Kubernetes.API.Auth.Provider.OIDC.GroupsClaim, "", "OIDC authorization provider GroupsClaim.")
+
```
Comment (pipo02mix) [inline main.go]: I dont see the installation flag wiring here, is it missing?

Comment (corest) [reply main.go]: It is rather should not be anywhere as it is not used anywhere.

**Review by rossf7 — APPROVED** [review]
LGTM apart from a nitpick on imports ordering.

**Review by pipo02mix — APPROVED** [review]
LGTM if it works for you

**Review by tuommaki — APPROVED** [review]
Given that it has been tested to work, LGTM.

**Pull request merged**